resources/assets/components/utilities/ProgressBar/ProgressBar.js

```diff
@@ -24,7 +24,7 @@ const ProgressBar = ({ completed, target, testId }) => {
 
   const label =
     percentCompleted > 100
-      ? `You're ${percentCompleted}% to your goal!`
+      ? `🎉 You're at ${percentCompleted}% of your goal! 🎉`
       : `${percentCompleted}% To Your Goal!`;
```

Comment:
Let's keep our capitalization consistent between these two messages
```suggestion
      : `${percentCompleted}% to your goal!`;
```